cfgov/unprocessed/css/organisms/header.less

```diff
@@ -83,6 +86,18 @@
             & > .m-global-search {
                 float: right;
             }
+
```

Comment:
Instead of nesting could you use the `m-global-header-cta__horizontal` to float the CTA?

Does Search even need the nesting? There's only one.